examples/github-team/README.md

```diff
@@ -0,0 +1,105 @@
+[<img src="https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg" width="400"/>][homepage]
+
+[![license][badge-license]][apache20]
+[![Terraform Version][badge-terraform]][releases-terraform]
+[![Join Slack][badge-slack]][slack]
+
+# Create a team which has access to two newly created repositories
+
+This example shows how to create a team and two repositories.
+The team will have pull permissions for one repository and push permissions
+for the other. We also add lists of members and maintainers to the team.
+
+## Basic usage
+
+The code in [main.tf] defines two public GitHub repository and two nested
+GitHub teams. The team will be granted pull permissions to the one repository
+and push permissions to the other. 
+
+```hcl
+# ---------------------------------------------------------------------------------------------------------------------
+# CREATE A TEAM AND TWO REPOSITORIES
+#
+# We create a team and two repositories. The team will have pull permissions for one repository and push permissions
+# for the other. We also add lists of members and maintainers to the team.
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "github_repository" "repository" {
+  name = "engineering-tools"
+}
+
+resource "github_repository" "another_repository" {
+  name = "devops-tools"
+}
+
```

Comment:
the readme should just copy the module usage itself not the full main.tf with all supporting resources..

```suggestion
```